src/style.css

```
@@ -0,0 +1,98 @@
+*{
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url("./fonts/Avenir-Book.ttc");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("./fonts/Avenir-Heavy.ttc");
+}
+
+.searchBar_wrapper{
+  width: 90%;
+  margin: auto;
+}
+
+.bigFormLabel,
+.smallFormLabel {
```

```suggestion
.label {
```
Краще створити один загальний клас для обох лейблів, де будуть прописані загальні стила, і окремо створити класи для великого і малого (так само і з інпутом і картинкою)

```
<label for="bigFormInput" class="label bigFormLabel">
```